Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView+ValidationErrorVisibility.swift

```diff
@@ -25,8 +25,10 @@ public extension FeatureFormView {
     
     /// Specifies the visibility of validation errors in the form.
     /// - Parameter visibility: The preferred visibility of validation errors in the form.
-    func validationErrors(_ visibility: ValidationErrorVisibility) -> some View {
```

Comment:
This needed to change to `-> FeatureFormView` so that the two `FeatureFormView` modifiers can be placed in any order.